test/check-machines-consoles

```diff
@@ -122,6 +122,10 @@ fullscreen=0
         m = self.machine
         name = "vmWithSerialConsole"
 
+        # Restrict number of remembered console card states to 1
+        self.machine.write("/etc/cockpit/machines.override.json",
+                           """{ "config": { "MaxConsoleCardStates": 1 } }""")
```

Comment:
Why? Doesn't that nullify the effect of "multiple terminals"? Also, we really need at least one test for the default config. The comment shouldn't explain the (obvious) "what", but the "why".

Reply:
Yes, good points.

> Doesn't that nullify the effect of "multiple terminals"?

Kinda, in order to test the LRU cache. But we should test with at least two states and three VMs, agreed.